packages/near-membrane-dom/src/window.ts

```diff
@@ -116,27 +106,14 @@ export function filterWindowKeys(keys: PropertyKey[], remapTypedArrays: boolean)
  * that will be installed (via the membrane) as global descriptors in
  * the red realm.
  */
-export function removeWindowDescriptors<T extends PropertyDescriptorMap>(
-    unsafeDescs: T,
-    remapTypedArrays: boolean
-): T {
+export function removeWindowDescriptors<T extends PropertyDescriptorMap>(unsafeDescs: T): T {
     // Remove unforgeable descriptors that cannot be installed.
     ReflectDeleteProperty(unsafeDescs, 'document');
     ReflectDeleteProperty(unsafeDescs, 'location');
     ReflectDeleteProperty(unsafeDescs, 'top');
     ReflectDeleteProperty(unsafeDescs, 'window');
     // Remove other browser specific unforgeables.
     ReflectDeleteProperty(unsafeDescs, 'chrome');
-    // Crypto and typed arrays must be from the same global object
-    if (remapTypedArrays === false) {
-        ReflectDeleteProperty(unsafeDescs, 'crypto');
-        ReflectDeleteProperty(unsafeDescs, 'Crypto');
-        ReflectDeleteProperty(unsafeDescs, 'SubtleCrypto');
-        ReflectDeleteProperty(unsafeDescs, 'Blob');
-        ReflectDeleteProperty(unsafeDescs, 'File');
-        ReflectDeleteProperty(unsafeDescs, 'FileReader');
-        ReflectDeleteProperty(unsafeDescs, 'URL');
-    }
```

Comment:
I'm just going to assume that this functionality will all return in some form in later commits?